packages/websocket/CMakeLists.txt

```diff
@@ -8,9 +8,10 @@ endif()
 
 find_package(soss-core REQUIRED)
 find_package(soss-json REQUIRED)
-find_package(websocketpp REQUIRED)
+find_package(yaml-cpp)
```

Comment:
This can be removed if we put `find_dependency(yaml-cpp)` into `soss-coreConfig.cmake`.